ITISFoundation/osparc-simcore
# 🐛 Add missing email env variables to webserver (:warning: DEVOPS)

## What do these changes do?
These changes add 3 email env variable that were missing after Webserver ENV Cleanup. The variables are:
* SMTP_USERNAME
* SMTP_PASSWORD
* SMTP_PROTOCOL

## Related issue/s
- fixes https://github.com/ITISFoundation/osparc-ops-environments/issues/319

## How to test
Run simcore locally and ensure it starts and `webserver_email` is initialized with the provided variables

## DevOps Checklist
* Make sure while deploying to production that webserver has SMTP_USERNAME, SMTP_PASSWORD AND SMTP_PROTOCAL variable setup.
